.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "env/bin/python3.7"
+}
```

Comment:
I think we should not have pushed this folder `.vscode`. It should be removed

Reply:
I have taken care of it in my code

Reply:
Thanks for that Burundi